app/scripts/lib/AbstractPetnamesBridge.ts

```diff
@@ -35,8 +35,8 @@ type ChangeList = Record<ChangeType, PetnameEntry[]>;
 type PetnamesBridgeAllowedEvents = NameStateChange;
 
 export type PetnamesBridgeMessenger<
-  Event extends EventConstraint,
-  Action extends ActionConstraint,
+  Event extends EventConstraint = PetnamesBridgeAllowedEvents,
```

Comment:
You want this to be `never`. The reason being that we are already taking the union of Event with PetnamesBridgeAllowedEvents on line 43, so this default is confusing.